packages/connect-examples/electron-main-process/package.json

```
@@ -53,7 +53,7 @@
         }
     },
     "dependencies": {
-        "@trezor/connect": "../../connect"
+        "@trezor/connect": "9.0.0-beta.1"
```

```suggestion
        "@trezor/connect": "beta"
```
maybe lets use "recent beta" instead?
we are already on beta-4? maintaining version in example packages will be pain in the ass :)
one we finally release connect 9 it could be changed to `latest`

what do you think?